odxtools/element.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import asdict, dataclass
+from typing import List, Optional
+from xml.etree import ElementTree
+
+from odxtools.exceptions import odxrequire
+from odxtools.utils import create_description_from_et
+
+from .odxlink import OdxDocFragment, OdxLinkId
+
+
+@dataclass
+class NamedElement:
+    short_name: str
+    long_name: Optional[str]
+    description: Optional[str]
+
+    @staticmethod
+    def _from_et(
+        et_element: ElementTree.Element,
+        doc_frags: List[OdxDocFragment],
+    ) -> "NamedElement":
+
+        return NamedElement(
+            short_name=odxrequire(et_element.findtext("SHORT-NAME")),
+            long_name=et_element.findtext("LONG-NAME"),
+            description=create_description_from_et(et_element.find("DESC")),
```

Comment:
I think that this is now the only user of `create_description_from_et()`? If so, the function should be made a private static method of the class?

Reply:
There is one extra user for it

Reply:
okay, let's leave it like it is for now...